### PR TITLE
BZ1923220 Adding stalld known issue

### PR DESCRIPTION
**Pull request by kquinn1204**
@ahardin-rh Adding a known issue associated with stalld that initiated lots of debate as to how to doc.

### CODE REVIEW

**Comment by kquinn1204** [comment]
@MarSik Can you check what I have written here about stalld. I gave a bit more content as you asked hopefully I ma not revealing too much information of a sensitive nature. https://deploy-preview-29297--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues

**Comment by kquinn1204** [comment]
@ahardin-rh Can you review https://deploy-preview-29297--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues "The stalld service triggers..."

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -752,6 +752,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 Before upgrading, follow the procedure as described in xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#upgrading-performance-addon-operator-configured-for-a-specific-namespace_cnf-master[Upgrading Performance Addon Operator when previously installed to a specific namespace].
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913826[*BZ#1913826*])
+* The `stalld` service triggers a bug in the kernel, which results in the node freezing. In order to work around this issue, Performance Addon Operator disables `stalld` by default. The fix impacts latency associated with DPDK based workloads, however the functionality will be restored once the kernel bug (link:https://bugzilla.redhat.com/show_bug.cgi?id=1912118[*BZ#1912118*]) is fixed.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:755]: the Performance Addon Operator

**Comment by ahardin-rh** [comment]
@kquinn1204 Just one minor suggestion from me. We'll be good to merge once we get the okay from @MarSik. I think your approach to this is good and that we listed a public bug instead of one marked private.

**Review by MarSik — APPROVED** [review]

**Comment by kquinn1204** [comment]
@ahardin-rh correction made. Thanks.

**Review by codyhoag — APPROVED** [review]

**Comment by codyhoag** [comment]
FYI @vikram-redhat

**Pull request merged**